.github/workflows/ci.yaml

```diff
@@ -12,6 +12,12 @@ jobs:
       fail-fast: true
       matrix:
         php: [8.1, 8.2, 8.3]
+        drupal-constraint: ['^10.3.5', '^11.0.5']
```

Comment:
How come we use precise patch versions? Normally it's only the minor added.

Reply:
Having a version string with patch version is quite common in /vendor/, but less so in Drupal core, it seems.

All it implies is that is the lowest tested version.
We allow 10.3.5, 10.3.6, 10.4.0, ..., but not 10.3.4.
Ofc it can happen that 10.3.4 fails and 10.4.0 fails but 10.3.5 is ok and 10.4.1 is ok, because a fix was ported to two branches. Specifying the 10.3.5 patch number does not enforce a patch number for 10.4.x.
So, we can omit it.